tinymapper.yaml

```diff
@@ -0,0 +1,13 @@
+name: tinymapper
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bedtools
+  - bowtie2
+  - deeptools
+  - macs2
+  - mawk
+  - minimap2
```

Comment:
Why adding minimap2 dependency? Is there any part of the workflow relying on it?

Reply:
Indeed idk why it was in my env, just removed it